src/pages/en/reference/integrations-reference.md

```diff
@@ -215,9 +223,10 @@ The address, family and port number supplied by the [NodeJS Net module](https://
 
 ### `astro:server:done`
 
-**Previous hook:** [astro:server:start](#astroserverstart)
+**Previous hook:** [`astro:server:start`](#astroserverstart)
```

Comment:
I noticed that `astro:server:start` became `astro:server` on the production deploy without a code fence around them. I guess it's being treated as a JSX directive by accident? In any case, code fences fix the problem 🤷